b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/idpSection/otherComponents/confirmAddRemoveLoginFlowModal.js

```diff
@@ -126,7 +126,8 @@ export default function ConfirmAddRemoveLoginFlowModal(prop) {
 
 /**
  * 
- * @returns When then WSO2IS_APP_NAME is not the correct applicaiton, it will show this section
+ * @returns When then `config.ManagementAPIConfig.SharedApplicationName` is not the correct applicaiton, 
+ * it will show this section
```

Comment:
Shall we have a `.` at the end. Check other related places as well.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/8b059f39358c5652ea99e473f4cf34e23ad95e0b